src/psycopmlutils/loaders/raw/load_coercion.py

```diff
@@ -42,72 +42,89 @@ def coercion(
         df = sql_load(sql, database="USR_PS_FORSK", chunksize=None, n=n)
 
         df.rename(
-            columns={"datotid_start_sei": "timestamp", "varighed": "value"},
+            columns={"datotid_start_sei": "timestamp", "varighed_timer_sei": "value"},
```

Comment:
Oh, this is nonstandard! I'd expect the function to have a boolean in "value", not a duration. Definitely wanna specify this in the docstring. 

Also, do you need duration? Might want it in the function name as well.